api/base/serializers.py

```diff
@@ -5,9 +5,11 @@
 from rest_framework.fields import SkipField
 from rest_framework import serializers as ser
 
+from framework.guid.model import Guid
 from website import settings
 from website.util.sanitize import strip_html
 from website.util import waterbutler_api_url_for
+from website.project.model import Node, Comment
```

Comment:
Can you move this to be above line 10? Within the subcategories of imports that PEP8 defines, I'm trying to encourage that imports be sorted (alphabetical). 

Basically, a lot of imports were being duplicated/aliased inconsistently, because there was no clear place to look and see if it was already imported. :)